docker-storage-setup.conf

```diff
@@ -1,7 +1,28 @@
+# By default no new volume and fs will be setup for docker root dir. Docker
+# will create /var/lib/docker/ and use underlying fs for storing image and
+# containers.
+# 
+# If one wants to carve out a separate logical volume for storing docker
+# metadata and images/containers/volumes, then set DOCKER_ROOT_VOLUME=yes
+#
+# This will try to create a logical volume, create filesystem on that volume
+# and mount on $DOCKER_ROOT_DIR.
+DOCKER_ROOT_VOLUME=yes
+
```

Comment:
By default we don't want to create docker root volume. So this should be.

DOCKER_ROOT_VOLUME=no